mkdocs.yml

```diff
@@ -35,6 +35,7 @@ nav:
     - api/schema.md
     - api/storage.md
   - Integrations:
+    - Arrow: integrations/delta-lake-arrow.md
     - pandas: integrations/delta-lake-pandas.md
```

Comment:
Shouldn't Pandas, also start with an uppercase?

Reply:
@ion-elgreco - pandas has a weird convention where it should always be lowercase, no matter what.